docs/stream/introduction.mdx

```diff
@@ -0,0 +1,343 @@
+---
+title: "Introduction"
+---
+
+import CodeTabs from "@theme/CodeTabs";
+import TabItem from "@theme/TabItem";
+
+# Svix Stream
+
+Svix Stream is an event streaming platform that enables services to reliably deliver events to multiple destinations through a single integrated API.
+
```

Comment:
Probably throughout this whole docs page: draw parallels/comparisons to Svix webhooks entities.

I think it can clarify things to people that know what webhooks are.

Reply:
I avoided most of these comparisons for now. I actually think it's best if we avoid comparisons with webhooks, as it might cause more confusion where semantics between the two platforms differ.

I also think the descriptions of the entities for streams should be able to stand on their own